engine/src/engine.rs

```diff
@@ -886,23 +892,35 @@ impl Engine {
             .collect()
     }
 
-    fn schedule_promise(promise: PromiseCreateArgs) {
-        #[cfg(feature = "log")]
-        sdk::log_utf8(
-            crate::prelude::format!(
-                "Call contract: {}.{}",
-                promise.target_account_id,
-                promise.method
-            )
-            .as_bytes(),
-        );
+    fn schedule_promise(promise: PromiseCreateArgs) -> u64 {
+        sdk::log!(&crate::prelude::format!(
+            "Call contract: {}.{}",
```

Comment:
@mfornet there are tests that depend on the specific format of this log. Do we use it in production as well? If not I will change to harmonize with other logs (ie underscores in the label and only space (no colon) as the separator with the value).

Reply:
Let's change them to fit the guideline